### PR TITLE
Fix reset projections and add replay script

### PR DESCRIPTION
**Pull request by Orkin**

### CODE REVIEW

File: scripts/history_replay.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    use Symfony\Component\Stopwatch\Stopwatch;
+    chdir(dirname(__DIR__));
+    // Setup autoloading
+    require __DIR__ . '/../vendor/autoload.php';
+    // Retrieve the container
+    $container = require __DIR__ . '/../config/container.php';
+    /** @var $dbalConnection Connection */
+    $dbalConnection = $container->get('doctrine.connection.default');
+    //Make sure that the read model tables are empty
+    $dbalConnection->exec('TRUNCATE TABLE ' . Table::USER);
```
Comment (prolic) [inline scripts/history_replay.php]: don't do this, it's the responsibility of the read model implementation.

File: scripts/history_replay.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    $dbalConnection->exec('TRUNCATE TABLE ' . Table::TODO);
```
Comment (prolic) [inline scripts/history_replay.php]: remove

File: scripts/history_replay.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    $dbalConnection->exec('TRUNCATE TABLE ' . Table::TODO_REMINDER);
```
Comment (prolic) [inline scripts/history_replay.php]: remove

File: scripts/history_replay.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    $stopWatch = new Stopwatch();
```
Comment (prolic) [inline scripts/history_replay.php]: not needed, it's just 3 method calls, takes less than a second.

File: src/Projection/User/UserReadModel.php [hunk 5]
```diff
@@ -80,7 +80,7 @@ public function delete(): void
     {
         $tableName = Table::USER;
 
-        $sql = "DROP TABLE $tableName;";
+        $sql = "DROP TABLE `$tableName`;";
```
Comment (prolic) [inline src/Projection/User/UserReadModel.php]: :+1:

File: scripts/history_replay.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+ * (c) 2015-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+declare(strict_types=1);
+/**
+ * Replay all events to regenerate the read model
+ */
+namespace {
```
Comment (prolic) [inline scripts/history_replay.php]: use proophessor-do's namespace, not the global one.

File: scripts/history_replay.php [hunk 7]
```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * This file is part of prooph/proophessor-do.
+ * (c) 2014-2017 prooph software GmbH <contact@prooph.de>
```
Comment (prolic) [inline scripts/history_replay.php]: it's 2018

**Comment by prolic** [comment]
Also php-cs-fixer complained about newly added script.

**Comment by Orkin** [comment]
@prolic fix ;)

**Comment by prolic** [comment]
Thanks a lot !!

**Pull request merged**